woocommerce/woocommerce-gutenberg-products-block
# Allow running e2e test individually

Fixes #6228 

### Notes

In #6228, we introduced performance testing. As a side effect, it was no longer possible to run an individual e2e test. This PR aims to fix this regression and allows running an individual e2e test again.

This fix follows a similar approach as Gutenberg, which is using the following two config files:
1. [jest.config.js](https://github.com/WordPress/gutenberg/blob/trunk/packages/e2e-tests/jest.config.js) for the e2e tests
2. [jest.performance.config.js](https://github.com/WordPress/gutenberg/blob/trunk/packages/e2e-tests/jest.performance.config.js) for the performance tests

### Testing

#### Automated Tests

### Manual Testing

1. Start Docker Desktop.
2. Run `npm run wp-env destroy && npm run wp-env start`.
3. Run `npm run test:performance` to ensure the performance tests are running.
4. Run `npm run test:e2e -- tax` to ensure an individual e2e test can be run.
5. Run `npm run test:e2e-dev -- tax` to ensure an individual e2e test can be run in dev mode.
6. Run `npm run test:e2e-dev-watch -- tax` to ensure an individual e2e test can be run in dev mode and watch for further adjustments.
